File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Ocso/OUTSANGU00findBoxToDoctorInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Ocso
{
    [Serializable]
	public class OUTSANGU00findBoxToDoctorInfo
	{
		private String _doctor;
		private String _doctorName;
		private String _contKey;

		public String Doctor
		{
			get { return this._doctor; }
			set { this._doctor = value; }
		}

		public String DoctorName
		{
			get { return this._doctorName; }
			set { this._doctorName = value; }
		}

		public String ContKey
		{
			get { return this._contKey; }
			set { this._contKey = value; }
		}

		public OUTSANGU00findBoxToDoctorInfo() { }

		public OUTSANGU00findBoxToDoctorInfo(String doctor, String doctorName, String contKey)
		{
			this._doctor = doctor;
			this._doctorName = doctorName;
			this._contKey = contKey;
		}

	}
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/ocso/OUTSANGU00findBoxToDoctorInfo.java
package nta.med.data.model.ihis.ocso;

public class OUTSANGU00findBoxToDoctorInfo {
	private String doctor;
	private String doctorName;
	private String contKey;

	public OUTSANGU00findBoxToDoctorInfo(String doctor, String doctorName,
			String contKey) {
		super();
		this.doctor = doctor;
		this.doctorName = doctorName;
		this.contKey = contKey;
	}

	public String getDoctor() {
		return doctor;
	}

	public void setDoctor(String doctor) {
		this.doctor = doctor;
	}

	public String getDoctorName() {
		return doctorName;
	}

	public void setDoctorName(String doctorName) {
		this.doctorName = doctorName;
	}

	public String getContKey() {
		return contKey;
	}

	public void setContKey(String contKey) {
		this.contKey = contKey;
	}

}
